SAS/Jonckheere-Terpstra Test.qmd

```diff
@@ -0,0 +1,106 @@
+------------------------------------------------------------------------
+
+title: "SAS Jonckheere-Terpstra Test"
+
+------------------------------------------------------------------------
```

Comment:
For quarto we use three dashes to denote the beginning and end of the file header
```suggestion
---
title: "SAS Jonckheere-Terpstra Test"
---

```

Reply:
This will fix the failed check btw

Reply:
updated